index.html

```diff
@@ -120,17 +127,27 @@
                 
                 document.getElementById('attachment-file').value = "";
                 $('#m').val('');
-                
+
                 return false;
             });
 
 
             //Typing indicator
             document.getElementById("m").addEventListener("keyup", function () {
                 console.log("#sending typing event for userId: " + userId + ", groupId: " + groupId);
-                window.Applozic.ALSocket.sendTypingStatus("1", "1", userId, groupId);
+                sendTypingStatus("1", "1", userId, groupId);
             }, false);
 
+            let changeStatusBtn = document.querySelector('#update-agent-status')
+            
+            changeStatusBtn.addEventListener('click', () => {
+                let currStatus = changeStatusBtn.getAttribute('data-status');
+                currStatus = Number(currStatus)
+                updateAgentStatus(currStatus ^ 1);
+                changeStatusBtn.setAttribute('data-status', currStatus ^ 1);
+                changeStatusBtn.innerText = !!currStatus ? "Go Online" : "Go Offline";
+            })

```

Comment:
i think we should not put this code here. manually online offline seems not required. will discuss over slack